docker-compose.yml

```diff
@@ -9,14 +9,15 @@ services:
       test: ["CMD", "curl", "-f", "http://localhost:8080/health/ping"]
```

Comment:
personally I would delete this container - I do not see the use case for running locally with the app in a container